src/main/java/duke/Ui.java

```diff
@@ -11,25 +18,37 @@ public class Ui {
     public Ui() {
     }
 
-    // Duke's self-intro
+    /**
+     * Our lovely Duke gives a sweet intro
+     */
     public void saysHi() {
```

Comment:
Might be better if the name of the method is written in imperative form (sayHi instead of saysHi, sayBye instead of saysBye)

Reply:
Indeed, I should improve on that (as well as the other comments you've made). Thank you for taking the time to go through this in detail and giving the good nuggets of feedback! Cheers!